pyaedt/downloads.py

```diff
@@ -32,15 +31,15 @@ def _get_file_url(directory, filename=None):
         return EXAMPLE_REPO + "/".join([directory, filename])
 
 
-def _retrieve_file(url, filename, directory, destination=None):
+def _retrieve_file(url, filename, directory, destination=None, local_paths=[]):
```

Comment:
I would suggest to not use list as default argument.
```suggestion
def _retrieve_file(url, filename, directory, destination=None, local_paths=None):
```